amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -1,94 +1,219 @@
+. Drag *On Deleted Object* to the Studio canvas.
+. On the *On Deleted Object* configuration screen, optionally change the value of the *Display Name* field.
+. Specify values for the following fields:
 +
-You can either enter your credentials into the global configuration properties or reference a configuration placed in a file in the `src` folder of your project file that contains these values. For simpler maintenance and better reuse of your project, use a configuration file.
+* *Bucket name* +
+Name of the bucket containing the multipart upload to abort
+* *Notification queue name* +
+Queue name used to notify users when a new object is created or deleted in an S3 bucket
+* *Scheduling Strategy* +
+Configures the scheduler that triggers the polling
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the *On Deleted Object* source in the app.
+. On the *General* tab, specify the connection information for the connector.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for Amazon S3, follow these steps:
+
+. In the *Mule Palette* view, select *Amazon S3* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Amazon S3 system. Amazon S3 supports basic and role authentication.
+
+To configure the global element for Amazon S3 Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, in *Connection*, select the authentication method to configure:
+  * <<basic-authentication,Basic Authentication>>
+  * <<role-authentication,Role Authentication>>
+
 +
-. If you are:
+You can reference a configuration file that contains ANT-style property placeholders (recommended), or you can enter your authorization credentials in the global configuration properties. For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```

Comment:
This should not be here.

Reply:
This seems to be in the right place according to the template.

Reply:
You're right Sathya. I think it should be moved because users can use variables for any field, but we haven't all agreed on that yet, so it's fine to leave it where you have it.